Helper/OmiseHelper.php

```diff
@@ -30,6 +31,37 @@ class OmiseHelper extends AbstractHelper
      */
     protected $header;
 
+    /**
+     * @var array
+     */
+    private $offsitePaymentMethods = [
+        Alipay::CODE,
+        Internetbanking::CODE,
+        Installment::CODE,
+        Truemoney::CODE,
+        Pointsciti::CODE,
+        Fpx::CODE,
+        Alipayplus::ALIPAY_CODE,
+        Alipayplus::ALIPAYHK_CODE,
+        Alipayplus::DANA_CODE,
+        Alipayplus::GCASH_CODE,
+        Alipayplus::KAKAOPAY_CODE,
+        Alipayplus::TOUCHNGO_CODE,
+        Mobilebanking::CODE,
+        Rabbitlinepay::CODE,
+    ];
+
+    /**
+     * Payment method payable by image code
+     *
+     * @var array
+     */
+    private $offsitePaymentImageCode = [
```

Comment:
Probably just terminology but normally payment methods with image codes are referred to as offlines rather than offsites. Or I guess we could rename it to just `paymentImageCode` to avoid confusion.

Reply:
Let's go with `offlinePaymentMethods` and drop the `ImageCode` if it makes sense. This will be consistent with the other variable name `offsitePaymentMethods.

Reply:
One thing to note though, there are also offline payment methods without image codes (like `Conveniencestore::CODE`), so may need a comment to clarify that this is only for offline payments with image code feature.